holoviews/plotting/bokeh/element.py

```diff
@@ -1241,6 +1241,10 @@ def _init_glyph(self, plot, mapping, properties):
         """
         Returns a Bokeh glyph object.
         """
+        try:
+            from bokeh.transform import Field
+        except ImportError:
+            from bokeh.core.property.vectorization import Field
```

Comment:
Why is this inlined?

Reply:
Was a quick test to see if it is the right thing. Bokeh 3 tests are now passing but bokeh 2 is failing pretty badly and it shouldn't be because the imports were inlined here.

Dunno if the Bokeh 2 test failures are related but I agree that these imports shouldn't be inlined once we got Bokeh 2 passing again.

Reply:
It *might* well be related, as in 2025b5d, bokeh 2 tests were *mostly* passing except for the extra fixes that worked in Bokeh 3 but made Bokeh 2 fail a lot more. Maybe these two `Field` classes aren't really interchangeable?

Reply:
Issue is moot now I am using `property_to_dict` and the tests are passing.